### PR TITLE
Refactors X and Y to Data & Column in XYAxis

### PR DESCRIPTION
**Pull request by anmoljagetia**
@GordonSmith Please Review.
Refactors XYAxis to have dataAxis and columnAxis instead of xAxis and yAxis.

### CODE REVIEW

File: src/chart/XYAxis.js [hunk 1]
```diff
@@ -63,18 +63,18 @@
 
         var test = element.append("g");
 
-        var svgXAxis = test.append("g")
+        var svgDataAxis = test.append("g")
```
Comment (GordonSmith) [inline src/chart/XYAxis.js]: This is not a "dataAxis" as it sometimes contains columns.  It _is_ an XAxis...

File: src/chart/XYAxis.js [hunk 2]
```diff
@@ -63,18 +63,18 @@
...
             .attr("class", this.orientation() === "horizontal" ? "x axis" : "y axis")
             .attr("transform", "translate(0," + height + ")")
-            .call(this.orientation() === "horizontal" ? this.xAxis : this.yAxis)
+            .call(this.orientation() === "horizontal" ? this.dataAxis : this.columnAxis)
         ;
-        var svgYAxis = test.append("g")
+        var svgColumnAxis = test.append("g")
```
Comment (GordonSmith) [inline src/chart/XYAxis.js]: Same for this:   This is not a "columnAxis" as it sometimes contains columns. It is an YAxis...

File: src/chart/XYAxis.js [hunk 3]
```diff
@@ -49,10 +49,10 @@
 
         this.svg = element.append("g");
         this.svgData = this.svg.append("g");
-        this.svgXAxis = this.svg.append("g")
+        this.svgDataAxis = this.svg.append("g")
```
Comment (GordonSmith) [inline src/chart/XYAxis.js]: This is "XAxis" as it is always horizontal.

File: src/chart/XYAxis.js [hunk 4]
```diff
@@ -49,10 +49,10 @@
...
             .attr("class", this.orientation() === "horizontal" ? "x axis" : "y axis")
         ;
-        this.svgYAxis = this.svg.append("g")
+        this.svgColumnAxis = this.svg.append("g")
```
Comment (GordonSmith) [inline src/chart/XYAxis.js]: This is This is "YAxis" as it is always vertical.

**Comment by GordonSmith** [comment]
You need to refactor the "x" and "y" in the CSS also.

**Comment by mlzummo** [comment]
I thought we were keeping X and Y axis?

**Comment by GordonSmith** [comment]
@mlzummo These charts can now swap orientation (think bar / column) the styles are directed at the "data values" and "columns" rather than the x / y axis, which is why we are renaming some of the variable / class names (Note:  there are still some true x / y variables which should remane).

**Comment by GordonSmith** [comment]
Looks good.

**Pull request merged**